src/NServiceBus.SagaAudit.Tests/APIApprovals.cs

```diff
@@ -1,17 +1,25 @@
-﻿using System.Runtime.CompilerServices;
-using NServiceBus.Features;
-using NServiceBus.SagaAudit.Tests;
+﻿using NServiceBus.Features;
 using NUnit.Framework;
+using Particular.Approvals;
 using PublicApiGenerator;
 
 [TestFixture]
 public class APIApprovals
 {
     [Test]
-    [MethodImpl(MethodImplOptions.NoInlining)]
     public void Approve()
     {
-        var publicApi = ApiGenerator.GeneratePublicApi(typeof(SagaAuditFeature).Assembly, excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" });
-        TestApprover.Verify(publicApi);
+        var publicApi = ApiGenerator.GeneratePublicApi(typeof(SagaAuditFeature).Assembly, excludeAttributes: new[]
+        {
+                "Particular.Licensing.ReleaseDateAttribute",
+                "System.Runtime.Versioning.TargetFrameworkAttribute"
+            });
+
+#if NETFRAMEWORK
```

Comment:
This isn't how you should implement this. You want both frameworks to use the same file, ensuring that you don't diverge the API between the two assemblies.

The previous version of this file was mostly correct. You shouldn't need the `ReleaseDateAttribute` at all.